Update backlog cleanup (#387)
DXHeroes/knowledge-base-content: practices/backlog_cleanup.md

**TL;DR**

Backlog Cleanup is important to keep a well-managed backlog. An ordered and short list of tasks, issues, and ideas results in a better overview of what's important, faster planning, and a cleaner scope. Periodically remove obsolete items from the backlog that have been created throughout the project lifecycle.

## What Is Backlog Cleanup  

Backlog Cleanup describes the process of managing tasks, issues, and ideas in your product backlog. You can reorder, rewrite, and delete backlog items. 

![Product backlog](/files/backlog-cleanup.jpg)
[Close-up sticky note with product backlog](https://www.freepik.com/free-photo/close-up-sticky-note-with-product-backlog_7816260.htm#query=backlog&position=0&from_view=search)

## Why You Might Want Backlog Cleanup

> Typically, companies have a project board, such as Trello, Jira, or GitHub projects. The boards contain columns for the different stages and the inbox or backlog column contains many different items. After some strategy meetings in the past months, some of the items in your backlog may be out of scope, since the product strategy or your team's priorities have shifted in a different direction.

Author: ```suggestion
Typically, companies have a project board, such as GitHub projects, that contains columns for different stages. The inbox or backlog column contains many different items. After some strategy meetings in the past months, some of the items in your backlog may be out of scope, since the product strategy or your team's priorities have shifted in a different direction.
```
